app/assets/stylesheets/components/_header.scss

```diff
@@ -1,6 +1,6 @@
 .site-header {
   align-items: center;
-  background-color: #76c261;
```

Comment:
Ah yep.

Reply:
no worries, figured it was just a missed debugging thing (like `border: solid 1px red;`)

Reply:
Ugh yeah I hate when that happens ;-)